game_api.txt

```diff
@@ -160,6 +160,28 @@ on_rotate(pos, node, user, mode, new_param2)
 ^ use on_rotate = screwdriver.disallow to always disallow rotation
 ^ use on_rotate = screwdriver.rotate_simple to allow only face rotation
 
+Sethome API
+--------------
+The Sethome API allows modders to use sethome's system in their own mods with the help of three functions :
+
+sethome.set_home(name)
```

Comment:
Should be `set_home(name, pos)`.  Pos can default to the player's current position if it's `nil` and they're online though.